landingai/predict.py

```diff
@@ -720,13 +726,13 @@ def _create_session(url: str, num_retry: int, headers: Dict[str, str]) -> Sessio
     retries = Retry(
         # TODO: make them configurable
         total=num_retry,
-        backoff_factor=3,
+        backoff_factor=3,  # This the amount of seconds to wait on the second retry (i.e. 0, 3, 6, 12). First retry is immediate
         raise_on_redirect=True,
         raise_on_status=False,  # We are already raising exceptions during backend invocations
         allowed_methods=["GET", "POST", "PUT"],
         status_forcelist=[
             # 408 Request Timeout , 413 Content Too Large, 500 Internal Server Error
-            429,  # Too Many Requests  (ie. rate limiter)
+            # 429,  # Too Many Requests  (ie. rate limiter). This is handled externally
```

Comment:
didn't quite get this part, how is the retry handled externally?
could you elaborate more about the line?